### ISSUE TITLE
Backup everything

### ISSUE THREAD

**Issue by sjelfull** [nystudio107/craft-scripts#12]
You probably want to backup database, assets and files at the same time, so probably make sense to have a script that runs all of them, and optionally sync them to S3 at the same time?

That means you don't have to keep 4 cron jobs for every site.

**Comment by khalwat** [comment]
Yeah, I left them as separate scripts to allow for flexibility, but you're right that in general, you will most often want to "backup all the things"

**Comment by terryupton** [comment]
@sjelfull I created a separate script called `sync` that contains these lines:
```
#!/bin/bash
bash pull_assets.sh
bash pull_db.sh
```

You can obviously add more commands if you want to backup/run other things.
But then I just cd into scripts and run `bash sync` and then it runs these together which is effectively running a single backup script.
